proto-lens-descriptors/proto-lens-descriptors.cabal

```diff
@@ -27,7 +27,7 @@ library
     , data-default-class >= 0.0 && < 0.2
     , lens-family == 1.2.*
     , lens-labels == 0.1.*
-    -- Specify an exact version of `proto-lens`, since it's tied closely
-    -- to the generated code.
-    , proto-lens == 0.2.2.0
+    -- Specify a more precise version of `proto-lens`, since it depends on
+    -- the exact definition of the Message class.
```

Comment:
For what it's worth, I do still prefer separate packages for `proto-lens-combinators/arbitrary/optparse/etc`.  It's annoying when there's a new major version of GHC, but otherwise they've been updated pretty independently.